src/code/Utils.cs

```diff
@@ -254,18 +254,62 @@ internal class ExtensionVaultModule
                 [string] $Command,
                 [hashtable] $Params
             )
-        
-            $module = Import-Module -Name $ModulePath -Passthru
+
+            $verboseEnabled = $Params.AdditionalParameters.ContainsKey('Verbose') -and ($Params.AdditionalParameters['Verbose'] -eq $true)
+            $module = Get-Module -Name ([System.IO.Path]::GetFileNameWithoutExtension($ImplementingModuleName)) -ErrorAction SilentlyContinue
```

Comment:
Do you want Ignore instead of SilentlyContinue or do you want any error added to $error here?

Reply:
We want to suppress this first error, because it is entirely possible for the module to not yet be loaded.  In this case we import the module and any error is reported at that point.

Reply:
I understand you want to suppress the error.  The difference between `SilentlyContinue` and `Ignore` is that `SilentlyContinue` will still put the error onto `$Error`.  Since this is handled internally, it seems we should use `Ignore` as user scripts sometimes checks `$Error`

Reply:
Oh, yes.  The error can be ignored.  I'll update.